### PR TITLE
Fix wrong livereload pattern for sheet file

### PR DESCRIPTION
**Pull request by cmnstmntmn**
Fix the pattern to allow live reloading to be triggered.

Initial issue was raised here:
https://github.com/liveview-native/liveview-client-swiftui/issues/1429

### CODE REVIEW

**Comment by bcardarella** [comment]
Thanks! Can you add this to the CHANGELOG. We are implementing this requirement for all Add/Change/Fix PRs. I just pushed to add the next `unreleased` format to the changelog. This should go under `Fixed` and I'll cherry-pick over to the 0.3 branch

**Comment by cmnstmntmn** [comment]
@bcardarella sure, raised this change here: https://github.com/liveview-native/live_view_native_stylesheet/pull/84

**Comment by bcardarella** [comment]
@cmnstmntmn please see the failing test run, you'll need to update the tests to reflect the change in the Live Reload pattern

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+* fix live reload pattern for the sheet file https://github.com/liveview-native/live_view_native_stylesheet/pull/83
```
Comment (bcardarella) [inline CHANGELOG.md:10]: This should go under `Fixed`

**Comment by bcardarella** [comment]
Thanks! 🎉

**Pull request merged**